src/Comments.jsx

```diff
@@ -7,6 +7,7 @@ import '../style/index.css';
 import { TextField, shapes } from '@cimpress/react-components';
 import CommentsClient from './CommentsClient';
 import { SERVICE_URL } from '../config';
+import { getSubFromJWT } from '../lib';
```

Comment:
It might be a little bit confusing when lib will contain both bablified code and committed code.